silq/meta_instruments/layout.py

```diff
@@ -1360,7 +1452,7 @@ def setup(self,
 
         # Create acquisition shapes
         trace_shapes = self.pulse_sequence.get_trace_shapes(
-            sample_rate=self.sample_rate, samples=self.samples())
+            sample_rate=self.sample_rate(), samples=self.samples())
```

Comment:
didn't you reverse the sample_rate() calling?

Reply:
I believe I converted it into a parameter, so it should now have ()